packages/js/product-editor/src/blocks/generic/text-area/block.json

```diff
@@ -0,0 +1,56 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "woocommerce/product-text-area-field",
+	"title": "Product textarea block",
+	"category": "woocommerce",
+	"description": "A text-area field for use in the product editor.",
+	"keywords": [ "products", "description" ],
+	"textdomain": "default",
+	"attributes": {
```

Comment:
`placeholder` isn't listed here, though it is supported.

We should also support (same as `woocommerce/product-text-field`):

- `tooltip`
- `required`
- `disabled`

Reply:
I've added the attributes and types, but probably we need to keep improving these attributes. Can we do it in a follow-up?

Reply:
I'm cool with a follow-up, but we shouldn't consider the (sorely missing any details 😞) issue closed/fixed until they are added.

Reply:
I have updated the pull request so that it doesnt close the issue when merged